docs/ref/settings.txt

```diff
@@ -1343,9 +1343,15 @@ See :ref:`Filtering error reports<filtering-error-reports>`.
 
 Default: ``'webmaster@localhost'``
 
-Default email address to use for various automated correspondence from the
-site manager(s). This doesn't include error messages sent to :setting:`ADMINS`
-and :setting:`MANAGERS`; for that, see :setting:`SERVER_EMAIL`.
+Default ``From:`` header value to use for various automated email
+correspondence from the site manager(s). This describes the email address (and
+optionally the name) of the author of the email message, to be displayed in the
+interface of email clients. If you wish to set both a name and an email
+address, use a value like the following:
+``'FirstName LastName <example@example.com>'``
```

Comment:
This adding feels a bit too verbose. I think it's enough to say that the format allowed for the setting should be valid for the email sending protocol that the service configures. Something like:

```suggestion
Default email address for automated correspondence from the site manager(s).
This address is used in the ``From:`` header of outgoing emails (except for
those reporting error messages sent to :setting:`ADMINS` and
:setting:`MANAGERS`; for that, see :setting:`SERVER_EMAIL`) and can take any
format valid in the chosen email sending protocol.
```